tasks/docker_tasks.py

```diff
@@ -54,8 +54,11 @@ def dockerize_test(ctx, binary, skip_cleanup=False):
 
     with open(f"{temp_folder}/Dockerfile", 'w') as stream:
         stream.write(
-            """FROM docker/compose:debian-1.29.2
+            """FROM ubuntu:20.04
+COPY --from=docker/compose-bin:v2.26.1 /docker-compose /usr/bin/compose
```

Comment:
I wonder if we should use this version: https://github.com/DataDog/datadog-agent/pull/30699/files
Or if we think that we should not get rate limited on that branch